day2FE/my-app/src/app/home/home.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, inject, signal, WritableSignal } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatButton } from '@angular/material/button';
+import { HttpClient } from '@angular/common/http';
+import { AuthDirective } from '../auth.directive';
+
+// response type for the showRandomDog method
+type DogResponse = {
+  message: string;
+  status: string;
+};
+
+@Component({
+  selector: 'app-home',
+  standalone: true,
+  imports: [CommonModule, MatButton, AuthDirective],
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+})
+export class HomeComponent {
+  // hardcoded logged-in state
+  public readonly loggedIn: WritableSignal<boolean> = signal(false);
```

Comment:
```suggestion
  public readonly loggedIn = signal<boolean>(false);
```
Even `public readonly loggedIn = signal(false);` would be fine and the compiler will infer its type as `WritableSignal<boolean>`